src/events.js

```diff
@@ -11,36 +10,29 @@ const send = require(`${__dirname}/util/send`);
  * @returns {Promise<void>}
 **/
 const messageUpdate = async (oldMessage, newMessage, object) => {
-	if (oldMessage === undefined || newMessage == undefined || !oldMessage.mentions || !newMessage.mentions) {
-		throw new Error('Expected parameters \'oldMessage\', \'newMessage\' at position 1, 2');
-	}
-	if(oldMessage.author.bot || oldMessage.mentions.members.size == 0 && oldMessage.mentions.roles.size == 0) {
-		return false;
-	}
-
-	let newArray = []; let oldArray = [];
-	oldMessage.mentions.members.forEach((member) => {
-		if(!member.user.bot && member.id != oldMessage.author.id) {
-			oldArray.push(`${member}`);
-		}
+	/*
```

Comment:
Documentation comments for syntax isn't required: the one thing developers are good at is google aha. If you could remove this comment, I'm happy to merge.

Reply:
oh.. okay
Sure I will remove it in a second
Thank you

Reply:
Done.. After this merge, I will push the README without emotes as another PR followed by the automation (including husky as @RaZeSloth suggested)
Thank you